src/libs/export.c

```diff
@@ -383,7 +383,7 @@ static void _validate_dimensions(dt_lib_export_t *d)
   uint32_t height = atoi(gtk_entry_get_text(GTK_ENTRY(d->height)));
   if(width > d->max_allowed_width || height > d->max_allowed_height)
   {
-    width = width > d->max_allowed_width ? dt_conf_get_int(CONFIG_PREFIX "width"): width;
+    width = width > d->max_allowed_width ? dt_conf_get_int(CONFIG_PREFIX "width") : width;
```

Comment:
Is a single space before `:` really changing something ?

Reply:
It seems to correct the behavior in #6464 for me... go figure. :-)

Reply:
Ok, I just tried a few more quit/start cycles, and in one case still had the reset-to-zero problem happen, so... maybe that wasn't the cause? If that's the case, sorry for the noise...

Reply:
I have never seen that and I've been working quite a lot on the export module recently (see print-size PR). Anyway, that's a reformatting, so I'll merge :)

Reply:
Ok, but it closed the issue, which should probably still be open... sorry! I'm really curious about what the actual cause is, but I need to go to sleep before I make any more dumb assumptions.